x-pack/plugins/ml/public/explorer/explorer.js

```diff
@@ -900,10 +907,44 @@ export const Explorer = injectI18n(injectObservablesAsProps(
       }
     }
 
+    applyFilterFromTable = (fieldName, fieldValue, action) => {
```

Comment:
Ah, good one. In this case I'm using `entity_cell.js` in places we need the filter icons. That component expects a `filter` function as a prop that it triggers on click for each of the icons. It's set up with that order of arguments. Since I'm using that component and passing `applyFilterFromTable` I'm keeping the arguments the same so it's consistent with how it's being used in other places.